Dockerfile

```
@@ -0,0 +1,116 @@
+# cache packages in layer
+COPY backend/package.json /backend/package.json
+COPY backend/package-lock.json /backend/package-lock.json
+RUN --mount=type=cache,target=/usr/src/app/.npm \
+    npm set cache /usr/src/app/.npm && \
+    npm ci
+# install
+COPY backend /backend
+
+FROM dp.apps.rancher.io/containers/nodejs:22
+
+LABEL org.opencontainers.image.title="SUSE Application Collection" \
+    org.opencontainers.image.description="Integrate the Application Collection into your development lifecycle" \
+    org.opencontainers.image.vendor="SUSE LLC" \
+    com.docker.desktop.extension.api.version="0.3.4" \
+    com.docker.extension.categories="kubernetes,utility-tools" \
+    com.docker.desktop.extension.icon="https://raw.githubusercontent.com/rancherlabs/application-collection-extension/refs/heads/main/assets/logo.svg" \
+    com.docker.extension.screenshots="[\
+        {\"alt\":\"Collection\", \"url\":\"https://raw.githubusercontent.com/rancherlabs/application-collection-extension/refs/heads/main/assets/01_collection.png\"}, \
+        {\"alt\":\"Application Details\", \"url\":\"https://raw.githubusercontent.com/rancherlabs/application-collection-extension/refs/heads/main/assets/02_application-details.png\"}, \
+        {\"alt\":\"Workloads\", \"url\":\"https://raw.githubusercontent.com/rancherlabs/application-collection-extension/refs/heads/main/assets/03_workloads.png\"}\
+    ]" \
+    com.docker.extension.detailed-description="\
+        Build and run cloud-native applications with SUSE's trusted, curated, and continuously updated application collection.\
+        <br />\
+        This extension helps you integrating the Collection into your local development environment by:\
+        <br />\
+        <ul>\
+            <li>Managing the authentication: docker, helm and kubernetes credentials are automatically configured</li>\
+            <li>Making apps plug&play: installs the workloads with a predefined set of values ready for local deployment</li>\
+            <li>Helping you stay up-to-date: detects application updates and helps you through the update process</li>\
+        </ul>\
+        <br />\
+        Usage:\
+        <br />\
+        <ol>\
+            <li>Have a target kubernetes cluster configured in your context</li>\
+            <li>Install the extension</li>\
+            <li><a href=\"https:\/\/docs.apps.rancher.io\/get-started\/authentication\/#create-a-personal-access-token\">Generate an access token</a> and configure the authentication</li>\
+            <li>Start deploying workloads</li>\
+        </ol>\
+        " \
+    com.docker.extension.publisher-url="https://apps.rancher.io/" \
+    com.docker.extension.additional-urls="[\
+        {\"title\":\"Product page\",\"url\":\"https:\/\/www.suse.com\/products\/rancher\/application-collection\"},\
+        {\"title\":\"Web application\",\"url\":\"https:\/\/apps.rancher.io\"},\
+        {\"title\":\"Documentation\",\"url\":\"https:\/\/docs.apps.rancher.io\"}\
```

Do we need to escape these?
```suggestion
        {\"title\":\"Product page\",\"url\":\"https://www.suse.com/products/rancher/application-collection\"},\
        {\"title\":\"Web application\",\"url\":\"https://apps.rancher.io\"},\
        {\"title\":\"Documentation\",\"url\":\"https://docs.apps.rancher.io\"}\
```

---

Done dc86dab